mackup/applications/storyist-3.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = storyist-3
+
+[configuration_files]
+Library/Preferences/.com.storyist.storyist.plist
+Library/Preferences/com.storyist.storyist.plist
```

Comment:
Can you finish the line with a newline character?

Reply:
Will do

Reply:
Done